samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.h

```diff
@@ -17,5 +17,30 @@
 
 #import "MenuTableViewController.h"
 
-@interface UILabelViewController : MenuTableViewController
+@interface UILabelViewController : UIViewController <UITextFieldDelegate>
+@property (nonatomic) UITextField* text;
```

Comment:
correct, that is already part of this VC.

---
In reply to: [105046901](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r105046901) [](ancestors = 105046901,105043984,105040618,105038372,104793428)